lib/orbit/lib/diffs.js

```diff
@@ -91,6 +91,10 @@ var diffs = function(a, b, options) {
               bi++;
             }
           }
+        } else if (a instanceof Date) {
+          if (a.getTime() === b.getTime()) return;
```

Comment:
That's also the reason I only check the type of a and not b (we already know they're the same).